masterdata/configuration/liquibase/liquibase.xml

```diff
@@ -487,7 +487,7 @@
             INSERT INTO inv_stockroom (name, creator, date_created, location_id, uuid)
             VALUES ('Hosakote UPHC StockRoom', 1, NOW(), @locationidHUPSC, uuid());
             INSERT INTO inv_stockroom (name, creator, date_created, location_id, uuid)
-            VALUES ('Hosakote Nama Clinic StockRoom', 1, NOW(), @locationidHNamaClinic, uuid());
```

Comment:
Dont modify existing liquibase entries, this will cause checksum errors

Reply:
May write a new liquibase for updates

Reply:
sure @mohan-13